src/presentation/wallet/home/index.tsx

```diff
@@ -91,7 +91,7 @@ const HomeView: React.FC<HomeProps> = ({
           <Balance
             assetHash={lbtcAssetHash}
             assetBalance={fromSatoshiStr(assetsBalance[lbtcAssetHash] ?? 0)}
-            assetImgPath="assets/images/liquid-assets/liquid-btc.svg"
+            assetImgPath={getAssetImage(lbtcAssetHash)}
```

Comment:
I think here was fine to hardcode

Reply:
I removed the entire `assets/images/liquid-assets` folder.
By using `lbtcAssetHash` is locked to L-BTC.